Doesn't build with path 0.6

```
[ 11 of 124] Compiling System.Process.Read ( src/System/Process/Read.hs, dist/build/System/Process/Read.o )

src/System/Process/Read.hs:429:20: error:
    Ambiguous occurrence ‘PathException’
    It could refer to either ‘Path.PathException’,
                             imported from ‘Path’ at src/System/Process/Read.hs:69:1-21
                          or ‘System.Process.Read.PathException’,
                             defined at src/System/Process/Read.hs:426:1

src/System/Process/Read.hs:430:15: error:
    Ambiguous occurrence ‘PathException’
    It could refer to either ‘Path.PathException’,
                             imported from ‘Path’ at src/System/Process/Read.hs:69:1-21
                          or ‘System.Process.Read.PathException’,
                             defined at src/System/Process/Read.hs:426:1
```

--- a/src/System/Process/Read.hs
+++ b/src/System/Process/Read.hs
@@ -423,11 +423,11 @@ getEnvOverride platform =
           mkEnvOverride platform
         . Map.fromList . map (T.pack *** T.pack)
 
-newtype PathException = PathsInvalidInPath [FilePath]
+newtype InvalidPathException = PathsInvalidInPath [FilePath]
     deriving Typeable
 
-instance Exception PathException
-instance Show PathException where
+instance Exception InvalidPathException
+instance Show InvalidPathException where
     show (PathsInvalidInPath paths) = unlines $
         [ "Would need to add some paths to the PATH environment variable \
           \to continue, but they would be invalid because they contain a "

